### ISSUE TITLE
Support ObjectKeyword type

### ISSUE THREAD

**Issue by lucas-rudd** [lukeautry/tsoa#379]
The current functionality for utilizing the object keyword in an interface results in an error being thrown:

interface example:
```
interface MyInterface {
  attribute1: object;
  attribute2: array;
  attribute3: AnotherInterfaceType;
}
```

the resulting error:
`Error: Unknown type: ObjectKeyword`

While, the expected behavior is to generate a swagger definition with the type set to the following

```
type: object
```

Support for the `object` keyword would allow for interfaces with a parameter of the type `object` type to be defined in their contract.

**Comment by dgreene1** [comment]
Solved in https://github.com/lukeautry/tsoa/pull/378

**Comment by dgreene1** [comment]
@lucas-rudd I just released version 2.4.4 which contains this work. Thank your for being a member of our community. I'd love to see more people using tsoa. If you wouldn't mind sharing with people on twitter, reddit, or [dev.to](https://dev.to)?

Also, there were a bunch of other enhancements. Check those out in: https://github.com/lukeautry/tsoa/releases/tag/v2.4.4
